packages/picker/src/Picker.js

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { PickerBase } from '@swc-uxp-internal/picker/src/Picker.js';
+import { html } from '@spectrum-web-components/base';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import pickerStyles from '@swc-uxp-internal/picker/src/picker.css.js';
+import chevronStyles from '@spectrum-web-components/icon/src/spectrum-icon-chevron.css.js';
+
+import uxpStyles from './uxp-picker.css.js';
+
+class UxpPickerBase extends PickerBase {
+    render() {
+        super.render();
+        return html`
+            <span
+                id="focus-helper"
+                tabindex="${this.focused || this.open ? '-1' : '0'}"
+                @focus=${this.handleHelperFocus}
+            ></span>
+            <div
+                aria-haspopup="true"
+                aria-controls=${ifDefined(this.open ? 'menu' : void 0)}
+                aria-expanded=${this.open ? 'true' : 'false'}
+                aria-labelledby="icon label applied-label"
+                id="button"
+                class="button"
+                @blur=${this.handleButtonBlur}
+                @pointerdown=${this.handlebuttonPointerdown}
+                @focus=${this.handleButtonFocus}
+                @click=${this.handleButtonClick}
+                @keydown=${{
+                    handleEvent: this.handleEnterKeydown,
+                    capture: true,
+                }}
+                ?disabled=${this.disabled}
+                tabindex="-1"
+            >
+                ${this.buttonContent}
+            </div>
+            ${this.renderMenu}
+        `;
+    }
+}
+
+export { UxpPickerBase as PickerBase };
+
+class UxpPicker extends UxpPickerBase {
+    constructor() {
+        super(...arguments);
+        this.handleKeydown = (event) => {
```

Comment:
please add the reason in PR description.

Reply:
Should it be keydown or keyup?

Reply:
its keydown, on the similar line to SWC code and i have tested this code works in PS